app/views/applications/edit.html.erb

```diff
@@ -10,11 +10,12 @@
       }
     ]
   } %>
-  <h1 class="remove-bottom-margin">
```

Comment:
Was there any CSS associated with this class?

Reply:
Nope, it perhaps came from `govuk_admin_template`

https://github.com/alphagov/govuk_admin_template/search?q=remove-bottom-margin&unscoped_q=remove-bottom-margin